.tekton/console-acm-215-push.yaml

```
@@ -0,0 +1,621 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stolostron/console?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "false"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch
+      == "release-2.15"
```

@Randy424 
```suggestion
      == "main"
```
For the current release, recall that we run PRs against `main`.

---

@Randy424 Ack, sorry - I commented this on push... it should be on pull!